manifests/params.pp

```diff
@@ -212,6 +213,13 @@
     $default_ssl_cert    = '/etc/ssl/certs/ssl-cert-snakeoil.pem'
     $default_ssl_key     = '/etc/ssl/private/ssl-cert-snakeoil.key'
     $ssl_certs_dir       = '/etc/ssl/certs'
+    if versioncmp($apache_version, '2.4') >= 0 {
```

Comment:
This line is causing the lint failure:

`metadata-json-lint metadata.json
manifests/params.pp - WARNING: top-scope variable being used without an explicit namespace on line 216`